### PR TITLE
Add model metadata that is default uneditable

### PR DESCRIPTION
**Pull request by jocelynsarah**
r? @mlmanapat 

This shows information from a `metadata.txt` file below the model notes and doesn't provide the option to edit it. Information about the model that you don't want to accidentally overwrite can be stored in `metadata.txt` and `notes.txt` can stay for comments written by humans.

### CODE REVIEW

**Comment by mlmanapat** [comment]
lgtm

File: topmodel/model_data.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 HISTOGRAM_FILE = 'histogram.json'
 NOTES_FILE = "notes.txt"
+METATDATA_FILE = "metadata.txt"
```
Comment (krivoire) [inline topmodel/model_data.py]: sorry for a post-merge nit but I think there's a typo here (`METATDATA`)

File: topmodel/model_data.py [hunk 2]
```diff
@@ -183,6 +184,10 @@ def save_data_frame(self, df):
             scores_path = os.path.join(self.model_path, SCORES_FILE)
             self.file_system.write_file(scores_path, f.getvalue())
 
+    def get_metadata(self):
+        metadata_path = os.path.join(self.model_path, METATDATA_FILE)
```
Comment (krivoire) [inline topmodel/model_data.py]: (typo)

File: topmodel/model_data.py (hunk 1 above)
Comment (jocelynsarah) [inline topmodel/model_data.py]: Post-merge typo discoveries always welcome! Now fixed.

**Pull request merged**